lib/faraday/better_retry.rb

```diff
@@ -33,8 +33,10 @@ def call(env)
           log(env, exception)
           retries -= 1
           retries_header(env, retries)
-          sleep sleep_amount(retries + 1)
-          retry
+          if (sleep_amount = calculate_sleep_amount(retries + 1, env))
```

Comment:
Where is `calculate_sleep_amount` defined? Don't see it.

Reply:
It's in the Faraday codebase.

Reply:
Perhaps it's better to do a full replace of the retry middleware - than to patch this.

Reply:
Ah, of course.